### PR TITLE
OCI: add conformance test github action

### PR DESCRIPTION
**Pull request by wy65701436**
Signed-off-by: Wang Yan <wangyan@vmware.com>

### CODE REVIEW

**Comment by milosgajdos** [comment]
@wy65701436 just wondering we should probably add the badge to `README`, too.

```
[![](https://github.com/<org>/<repo>/workflows/oci-distribution-conformance/badge.svg)](https://github.com/<org>/<repo>/actions?query=workflow%3Aoci-distribution-conformance)
```

File: .github/workflows/conformance.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    steps:
+      - name: set up docker
+        uses: docker-practice/actions-setup-docker@0.0.1
+        with:
+          docker_version: 18.09
+          docker_channel: stable
+
+      - uses: actions/checkout@v2
+        with:
+          path: src/github.com/docker/distribution
```
Comment (milosgajdos) [inline .github/workflows/conformance.yml:19]: this should be `distribution/distribution`

File: .github/workflows/conformance.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+      - name: start distribution server
+        working-directory: ./src/github.com/docker/distribution
```
Comment (milosgajdos) [inline .github/workflows/conformance.yml:22]: Same as above

**Review by milosgajdos — APPROVED** [review]
LGTM

**Review by waynr — APPROVED** [review]
:+1:

**Pull request merged**